relay-server/src/actors/processor.rs

```diff
@@ -1832,10 +1832,18 @@ impl EnvelopeProcessorService {
                 let source = event.get_transaction_source();
 
                 metric!(
-                    counter(RelayCounters::EventTransactionSource) += 1,
-                    source = &source.to_string(),
-                    sdk = envelope.meta().client_name().unwrap_or("proprietary"),
-                    platform = event.platform.as_str().unwrap_or("other"),
+                    counter(RelayCounters::EventTransaction) += 1,
+                    source = match &source {
+                        TransactionSource::Other(_) => "other",
+                        source => source.as_str(),
+                    },
+                    platform =
+                        PlatformTag::from(event.platform.as_str().unwrap_or("other")).as_str(),
+                    contains_slashes = if event.transaction.as_str().unwrap_or("").contains('/') {
```

Comment:
Will `unwrap_or_default()` work here? Which also should give you an empty string. 

```suggestion
                    contains_slashes = if event.transaction.as_str().unwrap_or_default().contains('/') {
```

Reply:
That's more characters 😉, but yeah makes sense!